bin/authorizeuser.py

```diff
@@ -32,20 +31,22 @@ def irods_user_create(username):
     with iRODSSession(host=IROD_BASE_URL, port=1247, user=IROD_ADMIN_USN, password=IROD_ADMIN_PASS, zone=IROD_ZONE) as session:
         #Check for user already existing in IRODS itself. If so, Save the user in the local database and continue
         try:
-            session.users.get(username)
-        except (NoResultFound,UserDoesNotExist) as e:
-            result = session.users.create(username,'rodsuser')
-        else:
-            print("User already exists")
-        user_id = session.users.get(username)
-        u = IrodAuthorizedUser(user=username,uid=user_id.id)
-        u.save()
+            user_id = session.users.get(username)
+            print(f"User {user_id} already exists.")
+            u = IrodAuthorizedUser(user=username,uid=user_id.id)
+            u.save()
+        except (NoResultFound,UserDoesNotExist):
+            newUser = session.users.create(username,'rodsuser')
+            print(f"User {newUser} was created and will be saved.")
+            u = IrodAuthorizedUser(user=username,uid=user_id.id)
+            u.save()
+
```

Comment:
```suggestion
            user = session.users.get(username)
        except (NoResultFound,UserDoesNotExist) as e:
            user = session.users.create(username,'rodsuser')
        else:
            print("User already exists")
            user = session.users.get(username)
        u = IrodAuthorizedUser(user=username,uid=user.id)
        u.save()
```
The way you have it currently, it will log `User {user_id} already exists.` every time it queries and finds a user, but I don't think that's a good log message. Plus you're duplicating some lines that don't need to be and didn't keep the case where there's an exception other than the two stipulated. In the suggested edit I also changed the var name to `user` instead of `user_id` because that's what `users.get()` returns, not an ID.

Reply:
The comment above the logic `#Check for user already existing in IRODS itself. If so, Save the user in the local database and continue` is why I kept that log. Prior to this section a user is added, so this is supposed to validate the previous action. Arguably, it should fail there if a user does not exist but they keep it going...

Reply:
I agree I did not do this justice. I'm not a fan of indentions in python meaning separate code blocks - completely misread that "else statement. My reason for touching this code was to remove the unused "e" and I believe that is all that should be done here honestly. 

```suggestion
             #Check for user already existing in IRODS itself. If so, Save the user in the local database and continue
        try:
            session.users.get(username)
        except (NoResultFound,UserDoesNotExist):
            result = session.users.create(username,'rodsuser')
        else:
            print("User already exists")
        user_id = session.users.get(username)
        u = IrodAuthorizedUser(user=username,uid=user_id.id)
        u.save()
```